service/src/test/java/de/adorsys/psd2/sandbox/xs2a/testdata/TestDataServiceTest.java

```diff
@@ -216,6 +216,25 @@ public void getAccountsTestWithUnkonwnPsu() {
     assertEquals(account, Optional.empty());
   }
 
+  @Test
+  public void getAccountsForBankOfferedConsentTestSuccessful() {
+    final String giroAccountId = "9b86539d-589b-4082-90c2-d725c019777f";
+    final String savingsAccountId = "d460057b-053a-490a-a36e-c0c8afb735e9";
+
+    List<Account> accounts = testDataService.getAccountsForBankOfferedConsent();
+
+    if (accounts.isEmpty()) {
+      fail();
+    }
+
+    assertEquals(accounts.size(), 2);
```

Comment:
till now we mostly used `assertThat(accounts.size(), equalTo(2))`

Reply:
done